demo/stubs/api.json

```diff
@@ -3574,5 +3574,4415 @@
        "type":"CCD Print Type",
        "description":"Printing for CCD"
     }
+ ],
+ "jurisdictions": [
```

Comment:
This is part of a future task in this epic: https://tools.hmcts.net/jira/browse/RDM-3918

Reply:
Done